README.md

```diff
@@ -257,7 +267,7 @@ const main = ([p, s]: [parameter,storage]): ret =>
      });
 ```
 
-Explanations :
+Explanations:
 
 - `mint` function will allow you to create a unique NFT. You have to declare the name, description, symbol and ipfsUrl for the picture to display
 - to simplify, we don't manage the increment of the token_id here it will be done by the front end later. We encourage you to manage this counter onchain to avoid overriding and existing nft. There is no rule to allocate a specific number to the token_id but people increment it from 0. Also, there is no rule if you have a burn function to reallocate the token_id to a removed index and just continue the sequence from the greatest index.
```

Comment:
```suggestion
- to simplify, we don't manage the increment of the `token_id` here it will be done by the front end later. We encourage you to manage this counter on chain to avoid overriding and existing NFT. There is no rule to allocate a specific number to the `token_id` but people increment it from 0. Also, there is no rule if you have a burn function to reallocate the `token_id` to a removed index and just continue the sequence from the greatest index.
```